Makefile

```diff
@@ -121,9 +121,9 @@ CONFIG_RTW_SDIO_PM_KEEP_POWER = y
 ###################### MP HW TX MODE FOR VHT #######################
 CONFIG_MP_VHT_HW_TX_MODE = n
 ###################### Platform Related #######################
-CONFIG_PLATFORM_I386_PC = n
+CONFIG_PLATFORM_I386_PC = y
 CONFIG_PLATFORM_ARM_RPI = n
-CONFIG_PLATFORM_ARM_RPI3 = y
+CONFIG_PLATFORM_ARM_RPI3 = n
```

Comment:
If the default platform is changed here, we'll need to add a patch to meta-subscriber layer